src/converse-chatview.js

```diff
@@ -532,7 +542,7 @@
                      * See XEP-0085 Chat State Notifications.
                      */
                     _converse.connection.send(
-                        $msg({'to':this.model.get('jid'), 'type': 'chat'})
+                        $msg({'to':this.model.get('jid'), 'type': 'chat', 'from': _converse.connection.jid})
```

Comment:
I don't think this 'from' attribute is necessary because it gets added automatically by strophe.js AFAICR. You can remove it. Unless you have a clear reason why it's needed.